tests/playwright/assets/breakpoints.ts

```diff
@@ -33,6 +33,29 @@ export default class {
 		return [ 'mobile', 'tablet', 'desktop' ];
 	}
 
+	async saveOrUpdate( editor: EditorPage, toReload = false ) {
+		const hasTopBar: boolean = await editor.hasTopBar();
+		if ( hasTopBar ) {
+			const saveButton = "//button[text()='Save Changes']";
```

Comment:
@Svitlana-Dykun 
Can we reduce the nesting?

Reply:
I created two separate methods for saving changes with and without enabled Top Bar. Also, I moved them to editor-page.ts file, because they are not specific for breakpoints functionality.